### PR TITLE
Adjust sample code for notifications

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM fedora:29
+RUN dnf install -y npm git
+
+COPY . /frontend
+WORKDIR /frontend
+
+CMD ["rm", "-rf", "node-modules", "package.json"]
```
Comment (adamruzicka) [inline Dockerfile]: So when you run the container it wipes node-modules and package.json which is stored inside? Why?

Comment (bastilian) [reply Dockerfile]: :man_facepalming: Good catch! copied without thinking from [RedHatInsights/vulnerability-ui](https://github.com/RedHatInsights/vulnerability-ui/blob/master/Dockerfile#L5) though I wonder why it still works and installs packages without moaning.

Comment (adamruzicka) [reply Dockerfile]: Looking at the compose it seems you override the command there so it actually never triggers this line I guess.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1 @@
+LOCAL_CHROME_PATH=`$PWD`/../insights-chrome/build
```
Comment (adamruzicka) [inline .env.example]: Does js do some special kind of preprocessing or do the standard shell rules apply here?

Comment (bastilian) [reply .env.example]: the `.env` file, for which this is the example for, is only interesting for `docker-compose`, so shell rules.

Comment (adamruzicka) [reply .env.example]: Then this should be changed, compare the outputs of the following

```shell
echo "|$PWD|"
echo "|`echo $PWD`|"
echo "|`pwd`|"
echo "|`$PWD`|"
```

**Comment by adamruzicka** [comment]
Thanks @bastilian !

**Pull request merged**